<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="5" total="6" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-10T11:49:29 WAT" name="Default Suite" finished-at="2023-01-10T11:49:42 WAT" duration-ms="12647">
    <groups>
    </groups>
    <test started-at="2023-01-10T11:49:29 WAT" name="Native-Playwright" finished-at="2023-01-10T11:49:42 WAT" duration-ms="12647">
      <class name="tests.SoloRunner">
        <test-method is-config="true" signature="beforeTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:tests.SoloRunner@57d7f8ca]" started-at="2023-01-10T11:49:29 WAT" name="beforeTest" finished-at="2023-01-10T11:49:32 WAT" duration-ms="2734" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[TESTEUR@gmail.com]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[douuuuuuu]]>
              </value>
            </param>
            <param index="3">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="createNewUserTest(java.lang.String,java.lang.String,java.lang.String)[pri:1, instance:tests.SoloRunner@57d7f8ca]" started-at="2023-01-10T11:49:33 WAT" name="createNewUserTest" data-provider="getRegistrationData" finished-at="2023-01-10T11:49:41 WAT" duration-ms="8774" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[polapola@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[douuuuuuu]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[douuuuuuu]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[L'utilisateur existe déjà]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: L'utilisateur existe déjà
at org.testng.Assert.fail(Assert.java:110)
at tests.SoloRunner.createNewUserTest(SoloRunner.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewUserTest -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:tests.SoloRunner@57d7f8ca]" started-at="2023-01-10T11:49:42 WAT" name="afterTest" finished-at="2023-01-10T11:49:42 WAT" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- tests.SoloRunner -->
    </test> <!-- Native-Playwright -->
  </suite> <!-- Default Suite -->
</testng-results>
